service/src/test/java/skills/intTests/SkillApprovalSpecs.groovy

```diff
@@ -705,6 +719,8 @@ class SkillApprovalSpecs extends DefaultIntSpec {
 
         def approvalsEndpointRes = skillsService.getApprovals(proj.projectId, 50, 1, 'requestedOn', false)
         List approvalItems = approvalsEndpointRes.data.sort({ it.userId })
+        println "rejected ${approvalItems[1]}"
+        println "rejected ${approvalItems[2]}"
```

Comment:
did you mean to leave these `println` statements?

Reply:
nope, removed!